filters/filter_17_TrackParam/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+  "filterId": 17,
+  "name": "AdGuard Tracking Parameters list",
+  "description": "Filter that enhances privacy by removing tracking parameters from URLs.",
+  "timeAdded": 1618479248382,
+  "homepage": "https://kb.adguard.com/general/adguard-ad-filters#removeparam-filter",
+  "expires": "4 days",
+  "displayNumber": 1,
+  "groupId": 2,
+  "subscriptionUrl": "https://filters.adtidy.org/windows/filters/17.txt",
+  "tags": [
+    "purpose:privacy",
+  	"recommended"
```

Comment:
Let's not mark it as `recommended` just yet.

Reply:
removed this tag.